src/seedu/addressbook/AddressBook.java

```diff
@@ -1125,15 +1165,24 @@ private static String getUsageInfoForClearCommand() {
     }
 
     /**
-     * Builds string for showing 'view' command usage instruction
+     * Builds string for showing 'list' command usage instruction
      *
-     * @return  'view' command usage instruction
+     * @return  'list' command usage instruction
      */
-    private static String getUsageInfoForViewCommand() {
+    private static String getUsageInfoForListCommand() {
         return String.format(MESSAGE_COMMAND_HELP, COMMAND_LIST_WORD, COMMAND_LIST_DESC) + LS
                 + String.format(MESSAGE_COMMAND_HELP_EXAMPLE, COMMAND_LIST_EXAMPLE) + LS;
     }
-
+    
+    /**
```

Comment:
it is good to see you have updated the header comments; however, your tests haven't changed to include the new functionality.